inventory/hosts

```diff
@@ -178,6 +193,7 @@ au-staging
 uk-staging
 fr-staging
 es-staging
+br-staging
```

Comment:
same here, this group is for servers maintained by the global team.
this line can be deleted.

Reply:
This line needs to be removed :fire: